src/mlpack/bindings/markdown/print_docs.cpp

```diff
@@ -90,204 +139,237 @@ void PrintDocs(const std::string& bindingName,
   }
   cout << endl;
 
-  cout << doc.shortDescription << " ";
+  // Print short description for each language.
   for (size_t i = 0; i < languages.size(); ++i)
   {
-    cout << "[Detailed documentation](#" << languages[i] << "_"
-      << bindingName << "_detailed-documentation){: .language-detail-link #"
-      << languages[i] << " }";
+    BindingInfo::Language() = languages[i];
+    cout << "<div class=\"language-short-desc\" id=\"" << languages[i]
+        << "\" markdown=\"1\">" << endl;
+    if (addWrapperDocs[i])
+    {
+      string desc = replace_all_copy(
+          paramMethods[0].Doc().longDescription(), "|", "\\|");
+      cout << desc;
+    }
+    else
+    {
+      cout << doc.shortDescription << " ";
+      cout << "[Detailed documentation](#" << languages[i] << "_";
+      cout << bindingName << "_detailed-documentation){: .language-detail-link #"
+        << languages[i] << " }.";
+    }
+    cout << "</div>" << endl;
```

Comment:
I think we might need another `endl` here before the `</div>`.  The scripts expect a `</div>` on its own line (it makes finding it way easier...).

You can see the `</div>` ends up in the text for every language:

http://mlpack2.ratml.org/doc/mlpack-git/python_documentation.html#adaboost
http://mlpack2.ratml.org/doc/mlpack-git/cli_documentation.html#adaboost